tasks/publish-pyxis-repository/README.md

```diff
@@ -0,0 +1,17 @@
+# publish-pyxis-repository
+
+Tekton task to mark all repositories in the mapped snapshot as published in Pyxis.
+This is currently only meant to be used in the rh-push-to-registry-redhat-io pipeline,
+so it will convert the values to the ones used for registry.redhat.io releases.
+E.g. repository "quay.io/redhat-prod/my-product----my-image" will be converted to use
+registry "registry.access.redhat.com" and repository "my-product/my-image" to identify
+the right Container Registry object in Pyxis.
+
+
+## Parameters
+
+| Name               | Description                                                               | Optional | Default value        |
+|--------------------|---------------------------------------------------------------------------|----------|----------------------|
+| server | The server type to use. Options are 'production' and 'stage' | Yes | production |
```

Comment:
Can you make the spacing consistent? 13, 14 and 17 are done one way, 15 and 16 another

Reply:
Fixed. I'm not sure if it's worth the effort though ;)

Reply:
As long as it is consistent it doesn't matter to me, but I think consistency is good